tests/test_nn/test_representation.py

```diff
@@ -84,6 +82,8 @@ def _pre_instantiation_hook(self, kwargs: MutableMapping[str, Any]) -> MutableMa
             relation_representations_kwargs=dict(embedding_dim=self.dim),
             dims=self.dim,
         )
+        # inferred from triples factory
+        kwargs.pop("max_id")
```

Comment:
maybe it would be better to allow passing `max_id` and use this ID to check against the inferred one